src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getArea() {
```

Comment:
while @override annotation is technically optional it's better to use it to clearly indicate inherited methods and separate them from own ones